xchange-core/src/main/java/org/knowm/xchange/utils/BigDecimalUtils.java

```diff
@@ -8,6 +8,12 @@
 
   private BigDecimalUtils() {}
 
+  // Need this to avoid errors
+  // With ftx for lend it takes maximum 4 digit after the comma
```

Comment:
this should probably live in ftx adapters, or at the least have a more generic (not exchange specific) name here.

Reply:
Fixed